pulp_python/app/tasks/sync.py

```diff
@@ -173,44 +169,55 @@ def _parse_metadata(project, version, distribution):
     return package
 
 
-def _build_additions(additions, remote_metadata):
+def _build_additions(delta, remote_metadata):
     """
     Generate the content to be added.
 
+    Args:
+        delta (namedtuple): tuple of content to add, and content to remove from the repository
+        remote_metadata (list): list of contentunit metadata
+
     Returns:
-        generator: A generator of content to be added.
+        The PythonPackageContent to be added to the repository.
     """
-    for entry in remote_metadata:
-        if(entry['filename'] not in additions):
-            continue
-
-        url = entry.pop('url')
-        artifact = models.Artifact(md5=entry.pop('md5_digest'))
+    def generate():
```

Comment:
Is there a reason these changes are bundled together?

Reply:
Nope.  I can still split them apart but since I had done the refactor first, it was already there, and I didn't notice until after making these changes.

It really isn't a problem to split them up though if you want me to.

Reply:
Dont worry about it, just curious if there was a connection.